src/ui/components/statusrow/StatusRow.js

```diff
@@ -35,6 +35,12 @@ const getSafetyConformity = (entity) => {
     return safetyConformity[entity.userDefinedFields.udfchar30];
 };
 
+function processSafetyData(safetyData){
+    const [hasHazards, setHasHazards] = useState(false);
+    safetyData.then(datas => setHasHazards(datas.records !== '0'));
+    return hasHazards;
+}
+
```

Comment:
Also it's preferred to use await since it is more understandable. As you can see on [this link](https://stackoverflow.com/questions/54495711/async-await-vs-then-which-is-the-best-for-performance) it's the same but with less code.

Reply:
> Why are you using a state for the hasHazards?

I don't get the data directly but a promise instead. I need a way to return the data once I get the answer and this seems to be one of the simplest ways.

Reply:
> Also it's preferred to use await since it is more understandable. As you can see on [this link](https://stackoverflow.com/questions/54495711/async-await-vs-then-which-is-the-best-for-performance) it's the same but with less code.

I already tried with await but in this case I don't find a way to make it shorter since we can only use await in async functions.